Localide/Classes/LocalideMapApp.swift

```diff
@@ -106,9 +110,21 @@ public extension LocalideMapApp {
         }
         Localide.sharedManager.applicationProtocol.localideOpen(launchUrl, options: [:], completionHandler: completionHandler)
     }
+    
+    static func setupForAppleMapsFix(iosDisableAppleMapsLaunchFix: Bool) {
```

Comment:
Rename to use shouldUseAppleMapsUrlScheme or something similar
Feature flags are not relevant in this context